### PR TITLE
Migrate Integration of logs into Apache Kafka topic

### LINKED ISSUE #341: Kafka Log integration
# What should we add to our content collections?

Migrate Help article https://help.aiven.io/en/articles/4888136-kafka-logs-integration

# Is this something you would work on yourself?

[X] I plan to open a pull request for this

### PR DESCRIPTION
**Pull request by anelook**
Migrated article https://help.aiven.io/en/articles/4888136-kafka-logs-integration to resolve #341

### CODE REVIEW

File: docs/products/kafka/howto/integrate-service-logs-into-kafka-topic.rst [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+Prepare the destination to receive the logs
+-------------------------------------------
+
+In the Aiven for Apache Kafka overview page enable Kafka REST API and create a Kafka topic where you want to receive the logs.
+
+Add a new integration to the source
+-----------------------------------
+
+Go to the overview page of your PostgeSQL service, scroll the list of actions till you see **Service integrations** and click on **Manage Integrations**. A new pop up window will be displayed showing a list of available integrations for your service. Select **Kafka Logs** from this list.
```
Comment (lornajane) [inline docs/products/kafka/howto/integrate-service-logs-into-kafka-topic.rst:24]: Typo PostgeSQL (it can't pick up things it thinks are an acronym - but we could add it to the suggestions list)

File: docs/products/kafka/howto/integrate-service-logs-into-kafka-topic.rst [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+Test the integration
+--------------------
+
+Go to the destination service overview page. Open the destination topic you specified to send logs. Navigate to the page with messages and press **Fetch Messages** to see the log entries that were sent from your source service. Select **Decode from base64** to see messages in JSON format.
+
+Edit or remove the integration
+------------------------------
+
+If you want to edit or remove the integration, use **Manage Integrations** in the source service. You'll see the created destination service in the list of **Enabled service integrations**, you can edit or remove it from here.
```
Comment (ftisiot) [inline docs/products/kafka/howto/integrate-service-logs-into-kafka-topic.rst:36]: I would rephrase 
> You'll see the created destination service in the list of **Enabled service integrations**, you can edit or remove it from here.
to 
> The kafka-logs integration is listed in the **Enabled service integrations** section, from where you can edit or remove it.

**Review by ftisiot — CHANGES_REQUESTED** [review]
Good work! Minor comments!

File: .github/styles/Aiven/common_replacements.yml [hunk 3]
```diff
@@ -6,6 +6,7 @@ swap:
   timeseries: time series
   kakfa: Kafka
   postgres: PostgreSQL
+  PostgreSQL: PostgreSQL
```
[the remark this answers is not in the document]
Comment (anelook) [reply .github/styles/Aiven/common_replacements.yml:9]: Oh no... I even tested that it pointed me to the error!

**Comment by anelook** [comment]
Thank you! Agree with the suggestions, added them.

File: docs/products/kafka/howto/integrate-service-logs-into-kafka-topic.rst [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+Setup Apache Kafka to receive the logs
+---------------------------------------
+
+Verify that Kafka REST API is enabled and create a Kafka topic where you want to receive the logs.
+
+Add a new integration to the source service
+-------------------------------------------
+
+Navigate to the Aiven console overview page of the source PostgreSQL service, scroll the list of actions till you see **Service integrations** and click on **Manage Integrations**. A new pop up window will be displayed showing a list of available integrations for your service. Select **Kafka Logs** from this list.
```
Comment (ftisiot) [inline docs/products/kafka/howto/integrate-service-logs-into-kafka-topic.rst:24]: Should this be a list of points to follow? Would make it easier to read and follow

**Review by ftisiot — CHANGES_REQUESTED** [review]
I would move the main Setup paragraph to a list of points to follow, in line with the other pages we have

File: docs/products/kafka/howto/integrate-service-logs-into-kafka-topic.rst [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+
+    Note, the integration can be used for both Aiven for Apache Kafka, as well as external Kafka clusters registered in the project's service integrations page. Read more on :doc:`how to manage Aiven internal and external integrations </docs/tools/cli/service/integration>`.
+
+In this example we'll show how to send the logs from an Aiven for PostgreSQL service to a topic in your Aiven for Apache
+Kafka service. For this example you need:
+
+-  a running Aiven for PostgreSQL service (we'll refer to it as *a source*)
+-  a running Aiven for Apache Kafka service (we'll refer as *a destination*)
+
+Setup Apache Kafka to receive the logs
```
Comment (lornajane) [inline docs/products/kafka/howto/integrate-service-logs-into-kafka-topic.rst:16]: Set up is two words when you are setting something up. If you are talking about your shiny new setup (a noun) it's one word.

File: docs/products/kafka/howto/integrate-service-logs-into-kafka-topic.rst [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+1. Navigate to the Aiven console overview page of the source PostgreSQL service.
+2. Scroll the list of actions till you see **Service integrations**.
+3. Click on **Manage Integrations**. A new pop up window will be displayed showing a list of available integrations for your service.
+4. Select **Kafka Logs** from this list.
+5. Select the destination Kafka service (or external Kafka integration) and the topic where the logs should be pushed.
+
+Test the integration
+--------------------
+
+1. Go to the destination service overview page.
```
Comment (ftisiot) [inline docs/products/kafka/howto/integrate-service-logs-into-kafka-topic.rst:33]: Should we state that these steps can be done only if your target Kafka is within Aiven? (and if it's an external kafka just check that the topic receives data)

**Review by ftisiot — CHANGES_REQUESTED** [review]
One small comment, almost there!

**Review by ftisiot — APPROVED** [review]
Good work!
